src/components/Warehouse/WarehouseDetailsContainer.scss

```diff
@@ -0,0 +1,133 @@
+@use "../../styles/partials/variables" as *;
+@use "../../styles/partials/mixins" as *;
+
+
+.detail-card-container {
+  width: 336px;
+  margin: auto;
+  position: relative;
+  top: -70px;
+  box-shadow: 0px 2px 5px 0px #13182c1a;
+  background: $color-white;
+  border-radius: 10px;
+  padding-top: 10px;
+  padding-left: 30px;
+  padding-right: 30px;
+  padding: 10px 30px 0 30px;
+  @include tablet {
+    width: 690px;
+    padding: 10px 0 0 0;
+  }
```

Comment:
THis is looking very similar to cardContainer so you should have moved it into a reusable block or mixin and extended it from here.